src/iperf_api.c

```diff
@@ -953,11 +954,19 @@ iperf_on_connect(struct iperf_test *test)
     struct sockaddr_in *sa_inP;
     struct sockaddr_in6 *sa_in6P;
     socklen_t len;
+    time_t now_secs;
```

Comment:
This duplicates the declaration of `now_secs` in line 948. However we do need a declaration of `struct iperf_time now;` somewhere in this function.